### PR TITLE
Vertex AI Experiments - Compare pipeline runs - Notebook review 

### PR DESCRIPTION
**Pull request by inardini**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

**Comment by inardini** [comment]
Ready to be reviewed. As [this PR](https://github.com/GoogleCloudPlatform/vertex-ai-samples/pull/853), the linter code modifies both this notebook and the `comparing_local_trained_models.ipynb`. Any ideas?

File: notebooks/official/experiments/comparing_local_trained_models.ipynb [hunk 1]
```diff
@@ -398,9 +398,14 @@
         "3. In the **Service account name** field, enter a name, and\n",
```
Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>H1 title missing above links (in same cell)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>Split into 4 separate sections</p><p>Move Dataset after Objective.</p><p>Add missing "uses" (see template) in Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>merge the two text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>merge the pip installs into above code cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>merge two text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>use notebook template for PROJECT_ID</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>merge 2 cells</p><p>remove 'Skip this step'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>use latest tempate</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>A bit more detail on what the next 2 code cells are doing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>merge the cleanup code cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>Set bucket to False</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

File: notebooks/official/experiments/comparing_pipeline_runs.ipynb [hunk 2]
```diff
@@ -66,6 +66,15 @@
     {
```
Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_pipeline_runs.ipynb:66]: <p>Move heading to above links (in links cell)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_pipeline_runs.ipynb:66]: <p>merge 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_pipeline_runs.ipynb:66]: <p>remove future tense: will</p><p> </p><p>eg.,</p><p>Next, you build the pipeline .....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_pipeline_runs.ipynb:66]: <p>consolidate into a single cleanup cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Comment (inardini) [reply notebooks/official/experiments/comparing_pipeline_runs.ipynb:66]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_pipeline_runs.ipynb:66]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_pipeline_runs.ipynb:66]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_pipeline_runs.ipynb:66]: <p>done</p>

**Comment by inardini** [comment]
Hi @andrewferlitsch, as I mentioned above, the linter modifies both `comparing_pipeline_runs` and the `comparing_local_trained_models` notebooks. But the one to review is only the 1st one. I will resolve all your comments on the 2nd one because I have already implemented in the #852 PR you closed yesterday. And I will resolve conflicts. Thanks

**Review by andrewferlitsch — APPROVED** [review]

**Comment by inardini** [comment]
Ready to be merged

**Pull request merged**